### PR TITLE
Ignore malformed

### PR DESCRIPTION
**Pull request by onesuper**
This small feature allow us to debug malformed data error

### CODE REVIEW

File: spark3/main.py [hunk 1]
```diff
@@ -86,17 +85,11 @@ def parse_trace_to_function(df: DataFrame,
 
         df.sql_ctx.udf.registerJavaFunction("decode_func_%s" % name, DECODE_CONTRACT_FUNCTION_UDF, schema)
 
-        if not contains_column(df.dtypes, "input", "string"):
-            raise ColumnNotFoundInDataFrame("input(string)", df)
```
Comment (onesuper) [inline spark3/main.py:82]: I remove the manual schema validation since DF's exception already contains rich info

**Review by TalkWIthKeyboard — APPROVED** [review]

**Comment by raycursive** [comment]
Since `spark3` object would be initialized on kernel starting, I'm wondering if adding setters for `default_abi_provider` and `ignore_malformed` (maybe including other properties such as tables and conditions) would be better for user overriding them?

**Review by raycursive — APPROVED** [review]

File: spark3/main.py [hunk 2]
```diff
@@ -62,6 +65,14 @@ def contract(self, address: str,
     def transformer(self):
         return self._transformer
 
+    @property
+    def ignore_malformed(self):
+        return self._ignore_malformed
+
+    @ignore_malformed.setter
```
Comment (raycursive) [inline spark3/main.py:72]: Oh I've seen the setter right here haha

Comment (onesuper) [reply spark3/main.py:72]: yep

Comment (onesuper) [reply spark3/main.py:72]: A hidden feature known by the admins

**Pull request merged**